app/models/book.rb

```diff
@@ -0,0 +1,6 @@
+class Book < ActiveRecord::Base
+  validates :title,           presence: true
```

Comment:
Part of me likes the verbosity here, but part of me likes the one-line solution, as well:

```ruby
validates_presence_of :title, :published_date, :author, :price
```